templates/admin/src/components/theme/action/Actions.tsx

```diff
@@ -39,26 +59,31 @@ export function ActionLink({
   );
 }
 
-export function ActionButton({
-  style, icon, cssClasses, onClick, isLoading, children,
-}: ActionButtonProps) {
+export function ActionButton(
+  {
+    style, icon, iconName, cssClasses, onClick, isLoading, children
+  }: ActionButtonProps
+) {
   return (
-    <div className={`action-container loading-button ${cssClasses}`}>
+    <div className={`action-container loading-button${cssClasses ? ' ' + cssClasses : ''}`}>
```

Comment:
l'espace avant plutot que de faire ' ' + ? (Mais je suppose que ça ne marche pas)

Reply:
why not, ca m'énerve juste quand je vois dans la console des espaces après les classnames quand la condition est pas remplie du coup je préfère faire dans ce sens la